src/src/pages/capabilities/KafkaCluster/NewTopicDialog.js

```diff
@@ -103,11 +107,15 @@ export default function NewTopicDialog({
 
   const handleAddClicked = () => {
     if (onAddClicked) {
+      let retention = formData.retention;
+      if (!isNaN(retention)) {
+        retention = `${retention}d`;
+      }
```

Comment:
similar comment, we need a fallback in case of NaN

Reply:
The idea in my head was if it is a NaN, then we don't know the format("but quite likely a string already") and shouldn't modify it.

Reply:
Got it now :)